frontend/fingertips-frontend/components/organisms/Inequalities/inequalitiesHelpers.test.ts

```diff
@@ -1356,3 +1345,23 @@ describe('getInequalitiesType', () => {
     );
   });
 });
+
+describe('reorderItemsArraysToEnd', () => {
+  it('Check that when specific headers are provided for reordering, the array is reordered accordingly.', () => {
+    const headers = reorderItemsArraysToEnd(
+      ['Age', 'Name', 'Sex', 'Other'],
+      ['Name', 'Age', 'Sex', 'Other']
+    );
+    expect(headers).toEqual(['Name', 'Age', 'Sex', 'Other']);
+  });
```

Comment:
I think if you're not keeping the existing test, it might be nice to have more items in the `headers` array than are in the `lastHeader` array. That way you're validating that items not specified are kept at the start of the array.

Something like:
```suggestion
  it('Check that when specific headers are provided for reordering, the array is reordered accordingly.', () => {
    const headers = reorderItemsArraysToEnd(
      ['Deprivation', 'Ethnicity', 'Age', 'Name', 'Sex', 'Other'],
      ['Name', 'Age', 'Sex', 'Other']
    );
    expect(headers).toEqual(['Deprivation', 'Ethnicity', 'Name', 'Age', 'Sex', 'Other']);
  });
```